Algorithm.Framework/Portfolio/IPortfolioOptimization.cs

```diff
@@ -0,0 +1,38 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+namespace QuantConnect.Algorithm.Framework.Portfolio.Optimization
+{
+    public enum ConstraintType { Equal = 0, Less = -1, More = 1 };
+
+    /// <summary>
+    /// Interface for portfolio optimization algorithms
+    /// </summary>
+    public interface IPortfolioOptimization
```

Comment:
Not sure if this is used, but please remove it and use the one defined on [master branch](https://github.com/QuantConnect/Lean/blob/master/Algorithm.Framework/Portfolio/IPortfolioOptimizer.cs) (the covariance can be calculated inside the implementations). Keep in mind someone setting their own [IPortfolioOptimizer](https://github.com/QuantConnect/Lean/blob/master/Algorithm.Framework/Portfolio/IPortfolioOptimizer.cs) from their algorithm - they'll just see the master interface.

Reply:
Thanks for noticing. I already switched to `IPortfolioOptimizer` but forgot to delete it. Now, it's removed.